src/index.tsx

```diff
@@ -341,15 +356,25 @@ class H5AudioPlayer extends Component<PlayerProps> {
             key={key}
             ref={this.progressBar}
             audio={this.audio.current}
+            srcDuration={this.props.useMSE.srcDuration}
             progressUpdateInterval={progressUpdateInterval}
             showDownloadProgress={showDownloadProgress}
             showFilledProgress={showFilledProgress}
+            onSeek={this.props.useMSE.onSeek}
```

Comment:
Need some null check so that it won't throw `srcDuration of undefined` error. Let's use destructuring above and give a default empty object.
```js
const {
  ...
  mse = {},
} = this.props
```
The other solution is to put default `{}` in `defaultProps`

Same for the other ones below